translator/ml_translatorLib.sml

```diff
@@ -2865,7 +2865,9 @@ val builtin_monops =
    Eval_empty_ffi,
    Eval_force_out_of_memory_error,
    Eval_Chr,
-   Eval_Ord]
+   Eval_Chrw,
+   Eval_Ord,
+   Eval_Ordw]
```

Comment:
I don't see why this, specifically, can't generate a Chr (WordToInt) at the level of primitives.  I am not talking about the basis.

Reply:
Ah I see what you mean now, so having `Char.chrw` translate to `Chr(WordToInt)` -- that makes sense, I misunderstood earlier.